pages/1.11/release-notes/1.11.1/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.11.1
+title: Release Notes for 1.11.1
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.11.1.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.0/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+
+DC/OS 1.11 includes many new capabilities, with a focus on:
+- Managing clusters across multiple clouds
+- Production Kubernetes-as-a-service
+- Enhanced data security
+- Updated data services
+
+Provide feedback on the new features and services at: [support.mesosphere.com](https://support.mesosphere.com).
+
+# Contents
+- [New Features and Capabilities](#new-features)
+- [Issues Fixed](#issues-fixed)
+- [Notable Changes](#notable-changes)
+
+<a name="new-features"></a>
+## New Features and Capabilities
+
+### Apache Mesos 1.5, Marathon 1.6, and Kubernetes 1.9 Integrated.
```

Comment:
Update this section for 1.11.1 and make the title more succinct.

Reply:
Done.